serverless/serverless
# Add resource count and warning to info display

## What did you implement:

This PR does two things:

1) When displaying Stack Info, it includes a `resources:` indication that shows the number of resources in the CloudFormation stack:

![resourcecount](https://user-images.githubusercontent.com/6509926/37248498-6bb86a38-2499-11e8-930e-963b7a81751b.png)

2) If there are 150+ resources in a service, it displays a warning that the user is approaching the 200 resource limit for CloudFormation and links them to a blog post on how to avoid it:
![resourcewarning](https://user-images.githubusercontent.com/6509926/37248501-893ddeb2-2499-11e8-931a-4484f1a911ac.png)

Hat tip to @Vadorequest for this idea initially. I thought it was a great idea. 💥 .

**I'll update the tests as needed but want to make sure we're good on the visuals first. Once those are settled, I'll make sure the tests pass.**



## How did you implement it:

As part of the `aws:info:gatherData` hook, I make a `listStackResources` API call for the CloudFormation stack and add it to the `gatheredData` object. Then I print it out in the `displayServiceInfo()` method.

## How can we verify it:

1) Run `sls info` in a deployed service. It should show `resources:` in the info. Similarly, it should show in the Service Information block after a deploy.

2) If you want to test the warning, set up a service with 30 HTTP functions and deploy it. It should display a warning.

## Todos:


***Is this ready for review?:*** NO
***Is it a breaking change?:*** NO
